drivers/gpio/Kconfig.stm32

```diff
@@ -9,6 +9,7 @@ menuconfig GPIO_STM32
 	bool "GPIO Driver for STM32 family of MCUs"
 	depends on GPIO && SOC_FAMILY_STM32
 	select HAS_DTS_GPIO
+	select USE_STM32_LL_GPIO
```

Comment:
You don't actually need that for now.
This is made to compile the ll .c files which are not used in the drivers for now (only functions defined in ll header files are used).